test/functional/test_framework/test_framework.py

```diff
@@ -1160,7 +1160,7 @@ def dynamically_prepare_masternode(self, idx, node_p2p_port, hpmn=False, rnd=Non
         platform_p2p_port = '%d' % (node_p2p_port + 101)
         platform_http_port = '%d' % (node_p2p_port + 102)
 
-        collateral_amount = 4000 if hpmn else 1000
+        collateral_amount = EVONODE_MASTERNODE_COLLATERAL if evo else MASTERNODE_COLLATERAL
```

Comment:
EVONODE_MASTERNODE_ sounds like node node.... May be just EVONODE_COLLATERAL ?

Reply:
I think EVOLUTION_MASTERNODE_* would work